src/views/labs/LabView/LabView.vue

```diff
@@ -31,18 +31,13 @@
       TagsPanel,
     },
   })
-  export default class LabView extends Vue {
-    data: LabViewData | null = null;
-
+  export default class LabView extends Mixins(PageDataMixin) {
     get lab() {
-      return this.data!.lab;
+      return (this.pageData as LabViewData)?.lab;
     }
 
-    async beforeRouteEnter(to: Route, from: Route, next: RouteCallback<LabView>) {
-      const data = (await axios.get(`/get/?type=project&nid=${to.params.nid}`)).data.data as LabViewData;
-      next((vm) => {
-        vm.data = data;
-      });
```

Comment:
Why did you remove it?
I think adding it back in the mixin would be fine though